cmake/ConfigFileSetting.cmake

```diff
@@ -210,7 +210,7 @@ nrn_check_symbol_exists("index" "" HAVE_INDEX)
 nrn_check_symbol_exists("isatty" "" HAVE_ISATTY)
 nrn_check_symbol_exists("iv" "" HAVE_IV)
 nrn_check_symbol_exists("lockf" "" HAVE_LOCKF)
-nrn_check_symbol_exists("mallinfo" "" HAVE_MALLINFO)
+nrn_check_symbol_exists("mallinfo" "" HAVE_MALLINFO2)
```

Comment:
Is this supposed to be still `mallinfo`? (or, `mallinfo2` ?)

Reply:
Good catch/ Thanks.